sdk/src/main/java/com/meniga/sdk/models/transactions/MenigaTransactionUpdate.java

```diff
@@ -0,0 +1,91 @@
+package com.meniga.sdk.models.transactions;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
```

Comment:
I think we could separate copyright and javadoc in the sdk files. Currently, `MenigaTransactionUpdate` class is described as:

```
Copyright 2017 Meniga Iceland Inc.
Created by agustk on 16.2.2018.

Returned by the transaction update method. Contains other transactions that might have the
same category as the updated transaction, in case the category was updated
```

If we generate javadoc files at some point or the sdk user looks at the class description within the IDE then the copyright statement is visible at the top instead of the meaningful description. We can fix it by using this scheme for our files:

```
/*
 * Copyright 2017 Meniga Iceland Inc.
 * Created by agustk on 16.2.2018.
 */
package com.meniga.sdk.models.transactions;

import ...

/**
 * Returned by the transaction update method. Contains other transactions that might have the
 * same category as the updated transaction, in case the category was updated
 */
public class MenigaTransactionUpdate implements Parcelable {
...
```
What do you think about it? (we could even add some gradle plugin like https://github.com/hierynomus/license-gradle-plugin that adds those copyright statements to the files)

Reply:
Sounds good to me.

Reply:
I'll be doing a documentation pass on the sdk soon where this can be updated.